src/firefly/js/templates/fireflyviewer/TriViewPanel.jsx

```diff
@@ -68,7 +68,7 @@ export const TriViewPanel= memo(( {showViewsSwitch=true, leftButtons, centerButt
                                            expandedMode={expanded===LO_VIEW.tables}/>);
     }
 
-    const isTriView = (showImages||coverageLeft) && (showXyPlots||coverageRight) && showTables;
+    const isTriView = (showImages||coverageLeft) && (showXyPlots||coverageRight) && showTablesInState;
```

Comment:
what is `showTablesInState`?

Reply:
that is how the parameter was passing in. I probably should get a better name.